src/lib/rest/ConnectionInfo.cpp

```diff
@@ -91,6 +92,13 @@ int uriParamOptionsParse(ConnectionInfo* ciP, const char* value)
     ciP->uriParamOptions[vec[ix]] = true;
   }
 
+  //
+  // Check of invalid combinations
+  //
+  if (ciP->uriParamOptions["keyValues"] && ciP->uriParamOptions["values"])   return -1;
```

Comment:
We are using the same "keyValues", "values", etc. in several parts. Using a #define for them will help to make the code easier to mantain.